Arm/rover_arm_system.hpp

```diff
@@ -76,5 +217,14 @@ class RoverArmSystem : public sjsu::common::RoverSystem
   void MoveElbow(){};
 
   void MoveWrist(){};
+
+  MissionControlData mc_data;
```

Comment:
```suggestion
  MissionControlData mc_data_;
```
I see drive system also has this incorrect @acarrou @ryerlic @Maaz0070